ClipboardMonitor/NativeMethods.txt

```diff
@@ -0,0 +1,12 @@
+AddClipboardFormatListener
+SetParent
+GetWindowText
+GetWindowTextLength
+GetForegroundWindow
+GetWindowThreadProcessId
+GetCurrentProcess
+
+// Below methods do not work properly with or does not exist in CsWin32
+//GetKernelObjectSecurity
+//SetKernelObjectSecurity
```

Comment:
As I mention elsewhere, I believe CsWin32 is in fact doing it properly. Just not the way you're used to (which also works, at least on the architectures and specific use cases you have tested).